app/sass/_icons.scss

```diff
@@ -1,5 +1,15 @@
 @import 'icons/index';
 
+.icon-minus {
```

Comment:
Rather than modifying the icon class, create a new class that can be applied in conjunction with the icon class. Make it dry.